src/modules/todoClass.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable */
```

Comment:
Please Remove the linter disabling command. It is good practice to follow the linter rules as the guidelines will help improve your code.